modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -1,70 +1,182 @@
-= Kubernetes Cluster Requirements
-:description: A list of requirements for provisioning Kubernetes clusters and worker nodes for running Redpanda in production.
+= Kubernetes Requirements and Recommendations
+:description: A list of requirements and recommendations for provisioning Kubernetes clusters and worker nodes for running Redpanda in production.
 :tags: ["Kubernetes"]
+:page-aliases: deploy:deployment-option/self-hosted/kubernetes/kubernetes-best-practices.adoc
 
-This topic is a checklist with the prerequisites and system requirements for installing production Redpanda in a Kubernetes cluster using the Helm chart.
+This topic provides you with the requirements and recommendations for provisioning Kubernetes clusters and worker nodes for running Redpanda in production.
```

Comment:
This topic provides requirements and recommendations for provisioning Kubernetes clusters and worker nodes for running Redpanda in production.